Dockerfile.in

```diff
@@ -1,10 +1,6 @@
 FROM alpine/helm:VERSION
-ARG AWS_REGION
-ARG AWS_ACCESS_KEY_ID
-ARG AWS_SECRET_ACCESS_KEY
 ENV HOME /tmp
 RUN helm init -c && \
     apk add --update --no-cache git bash && \
     helm plugin install https://github.com/hypnoglow/helm-s3.git && \
     rm -f /var/cache/apk/*
-RUN helm repo add infobloxcto s3://infoblox-helm-dev/charts
```

Comment:
This breaks an existing contract that many jobs depend on.

Reply:
I don't see args to prevent repo add from trying to sync s3, which is not possible without aws credentials. We could do it as a default entrypoint, the primary user of this image has already done something similar

Reply:
That's a good idea. Have a default entrypoint to add and sync the repo if it does not yet exist, using the AWS creds of the runtime environment.

Reply:
done